Resources/Private/Partials/Maps.html

```diff
@@ -59,6 +59,12 @@ <h6>{address.fullName}</h6>
                         <div>
                             <f:render partial="Address" section="address" arguments="{_all}"/>
                             <f:render partial="Address" section="contact" arguments="{_all}"/>
+                            <f:if condition="{settings.map.singlePid}">
+                                <f:link.action action="show" arguments="{address : address}"
+							        title="{address.title} {address.fullName}"
+                                    pageUid="{settings.map.singlePid}" class="btn">{f:translate(key:'more')} &hellip;
```

Comment:
I would remove the `&hellip;`, there is no need for that in the button

Reply:
I would remove class="btn" too. No need for it, link is IMHO sufficient. Isn't it?